pkg/odo/genericclioptions/context.go

```diff
@@ -79,10 +79,13 @@ func New(parameters CreateParameters, toggles ...bool) (context *Context, err er
 		}
 
 		// Parse devfile and validate
-		devObj, err := devfile.ParseDevfileAndValidate(parser.ParserArgs{Path: parameters.DevfilePath})
+		devObj, varWarning, err := devfile.ParseDevfileAndValidate(parser.ParserArgs{Path: parameters.DevfilePath})
 		if err != nil {
 			return context, fmt.Errorf("failed to parse the devfile %s, with error: %s", parameters.DevfilePath, err)
 		}
+
+		fmt.Printf("%#v\n", varWarning)
```

Comment:
Do you want to display it even if there are no warnings?

Reply:
oh, good catch. That was debugging that I forgot to delete :innocent: